pkg/services/ngalert/api/tooling/definitions/provisioning_contactpoints.go

```diff
@@ -6,7 +6,6 @@ import (
 	"github.com/grafana/alerting/alerting/notifier/channels"
 
 	"github.com/grafana/grafana/pkg/components/simplejson"
-	ngchannels "github.com/grafana/grafana/pkg/services/ngalert/notifier/channels"
 	"github.com/grafana/grafana/pkg/services/ngalert/notifier/channels_config"
```

Comment:
Following https://go.dev/blog/package-names, under_scores package names are to be avoided. Can you rename the package in a follow-up PR?